app/controllers/bulk_discounts_controller.rb

```diff
@@ -0,0 +1,12 @@
+class BulkDiscountsController < ApplicationController 
+
+  def index
+    @merchant = Merchant.find(params[:merchant_id])
+    @bulk_discounts = @merchant.bulk_discounts.all
```

Comment:
This could be more restful if I just call @merchant.bulk_discounts.each in the view, rather than this instance variable. I can refactor this next.